src/Microsoft.SqlTools.ServiceLayer/Agent/Notebooks/Exec-Notebook.ps1

```diff
@@ -0,0 +1,211 @@
+param(
+    [System.String]
+    $TemplateNotebook = $(throw New-Object System.Exception "TemplateNotebook cannot be null"),
```

Comment:
You should be able to just use [Parameter(Mandatory=$true)] to ensure that it has a value